packages/editor/src/components/block-inspector/index.js

```diff
@@ -37,13 +37,30 @@ const BlockInspector = ( { selectedBlock, blockType, count } ) => {
 		return <span className="editor-block-inspector__no-blocks">{ __( 'No block selected.' ) }</span>;
 	}
 
+	const hasSupportLink = blockType.support && blockType.support.url && blockType.support.label;
+
 	return (
 		<Fragment>
 			<div className="editor-block-inspector__card">
 				<BlockIcon icon={ blockType.icon } showColors />
 				<div className="editor-block-inspector__card-content">
 					<div className="editor-block-inspector__card-title">{ blockType.title }</div>
 					<div className="editor-block-inspector__card-description">{ blockType.description }</div>
+					{ hasSupportLink && (
+						<div className="editor-block-inspector__card-support">
+							{
+								blockType.support.external ? (
```

Comment:
Can you explain what makes a support link "external"? Is it based on the origin/hostname of the URL, and if so, is that not something we should programmatically determine on behalf of the block developer, than to impose on them?